pkg/kubectl/cmd/exec.go

```diff
@@ -69,7 +69,7 @@ func NewCmdExec(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *
 	}
 	cmd.Flags().StringVarP(&options.PodName, "pod", "p", "", "Pod name")
 	// TODO support UID
-	cmd.Flags().StringVarP(&options.ContainerName, "container", "c", "", "Container name. If omitted, the first container in the pod will be chosen")
+	cmd.Flags().StringVarP(&options.ContainerName, "container", "c", "", "Container name. If omitted, the first user container in the pod will be chosen")
```

Comment:
I tried to use `app container` at the beginning, but later I realize that this is not that clear, also @resouer said in other docs we use `user container`. I agree that we should keep first container here.